functional/platform/nspawn.go

```diff
@@ -395,6 +401,7 @@ UseDNS no
 			return
 		default:
 		}
+		log.Printf("Dialing machine: %s", addr)
```

Comment:
Sometimes when your environment has problems, this output message will help you to debug.